app/app.ts

```diff
@@ -42,10 +42,11 @@ function parseResult(result: any): { [key: string]: any } {
   // Most objects do not have DisableRankUpForMasterMode set, so don't include it unless it is set.
   result.disable_rankup_for_hard_mode = result.disable_rankup_for_hard_mode ? true : undefined;
   result.pos = [Math.round(result.data.Translate[0] * 100) / 100, Math.round(result.data.Translate[2] * 100) / 100];
+  result.korok = result.korok || undefined;
   return result;
 }
 
-const FIELDS = 'objid, map_type, map_name, map_static, hash_id, unit_config_name as name, `drop`, equip, data, messageid, scale, sharp_weapon_judge_type, hard_mode, disable_rankup_for_hard_mode, spawns_with_lotm, field_area';
+const FIELDS = 'objid, map_type, map_name, map_static, hash_id, unit_config_name as name, `drop`, equip, data, messageid, scale, sharp_weapon_judge_type, hard_mode, disable_rankup_for_hard_mode, spawns_with_lotm, field_area, korok_id as korok';
```

Comment:
Do we need the `as korok` here? If it's for the frontend I think it should just use "korok_id" instead of "korok".

Reply:
The FTS search term is currently `korok:` but  returns a `korok_id`.  If this is ok, I will make the change.

It is ok with me.

Reply:
Yeah that sounds fine to me.